.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Test
```

Comment:
I believe, it's easier to follow, if the filename and workflow name correspond.
E.g.
`build.yaml` - `build_and_test.yaml`
or
```suggestion
name: Build
```

Reply:
Addressed in commit 0b351127825f0dd446f0a0828295429ff5627f10 by renaming to test.yml.